Jenkinsfile

```diff
@@ -0,0 +1,24 @@
+pipeline {
+    agent {label 'slave'}
+
+    stages {
+        stage('run') {
+            steps {
+                bat 'ruby ex3.rb'
```

Comment:
you don't have any setup to do for this project so you could just add a simple echo here